src/services/transfer/__tests__/transfer-out-ehr-core.test.js

```diff
@@ -52,60 +51,58 @@ describe('transferOutEhrCore', () => {
       expect(sendCore).not.toHaveBeenCalled();
     });
 
-    it('should validate incomplete EHR where failed to retrieve EHR URL', async () => {
+    it('should stop EHR transfer if the requested EHR record does not exist in EHR repo', async () => {
       // when
       getRegistrationRequestStatusByConversationId.mockResolvedValueOnce(null);
       getEhrCoreAndFragmentIdsFromRepo.mockRejectedValueOnce(new EhrUrlNotFoundError());
+      patientAndPracticeOdsCodesMatch.mockResolvedValue(true);
 
-      const result = await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
+      await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
 
       // then
-      expect(result.inProgress).toBe(false);
-      expect(result.hasFailed).toBe(false);
       expect(createRegistrationRequest).toHaveBeenCalledWith(conversationId, messageId, nhsNumber, odsCode);
       expect(getEhrCoreAndFragmentIdsFromRepo).toHaveBeenCalledWith(nhsNumber, conversationId);
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        Status.MISSING_FROM_REPO
-      );
+      expect(updateConversationStatus).toHaveBeenCalledWith(conversationId, Status.MISSING_FROM_REPO);
       expect(logInfo).toHaveBeenCalledWith(`Getting patient health record from EHR repo`);
+      expect(logError).toHaveBeenCalledWith('EHR transfer out request failed', new EhrUrlNotFoundError());
       expect(sendCore).not.toHaveBeenCalled();
     });
 
-    it('should validate a failed download of the EHR from a presigned URL', async () => {
+    it('should stop EHR transfer if failed to download the EHR from S3 presigned URL', async () => {
       getRegistrationRequestStatusByConversationId.mockResolvedValueOnce(null);
       getEhrCoreAndFragmentIdsFromRepo.mockRejectedValueOnce(new DownloadError());
+      patientAndPracticeOdsCodesMatch.mockResolvedValue(true);
 
-      const result = await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
+      await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
 
-      expect(result.inProgress).toBe(false);
-      expect(result.hasFailed).toBe(false);
       expect(createRegistrationRequest).toHaveBeenCalledWith(conversationId, messageId, nhsNumber, odsCode);
       expect(getEhrCoreAndFragmentIdsFromRepo).toHaveBeenCalledWith(nhsNumber, conversationId);
-      expect(updateConversationStatus).toHaveBeenCalledWith(
-        conversationId,
-        Status.EHR_DOWNLOAD_FAILED
-      );
+      expect(updateConversationStatus).toHaveBeenCalledWith(conversationId, Status.EHR_DOWNLOAD_FAILED);
       expect(logInfo).toHaveBeenCalledWith(`Getting patient health record from EHR repo`);
+      expect(logError).toHaveBeenCalledWith('EHR transfer out request failed', new DownloadError());
       expect(sendCore).not.toHaveBeenCalled();
     });

```

Comment:
Similar to above, can we split this test up with given, when and then comments please? :)